tensorflow_datasets/core/visualization/audio_visualizer.py

```diff
@@ -0,0 +1,68 @@
+""" Audio Visualization
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+
+import random
+import IPython.display
+
+from absl import logging
+
+from tensorflow_datasets.core import dataset_utils
+from tensorflow_datasets.core import features as features_lib
+from tensorflow_datasets.core import lazy_imports_lib
+from tensorflow_datasets.core.visualization import visualizer
+plt = lazy_imports_lib.lazy_imports.matplotlib.pyplot
```

Comment:
> To check if a dataset has an audio feature, you can see on our catalog: For instance https://www.tensorflow.org/datasets/catalog/crema_d
> 
> It seems Groove is using `tfds.features.Tensor` instead of `tfds.features.Audio`, which sounds like a bug to me, we should upgrade groove to use audio feature. I'll open a bug for this:
> https://www.tensorflow.org/datasets/catalog/groove#groovefull-16000hz

Thank you for your comments - A bug for the Groove 'Tensor' attribute will be helpful - Should I report it as a bug ?

Reply:
The bug has been fixed so it should works if you are rebasing from master.

Reply:
It does, Thanks! The auto inferring works now ( [Link](https://colab.research.google.com/drive/1cK8De-bi3E87-LifRodsuAkvMpu1BLV-) . I have tested it with Groove, crema_d, ljspeech.